utils/oscap-chroot

```diff
@@ -74,26 +81,23 @@ function usage()
 }
 
 if [ $# -lt 1 ]; then
-    echo "No arguments provided."
-    usage
-    die
+    invalid "No arguments provided."
 elif [ "$1" == "-h" ] || [ "$1" == "--help" ]; then
     usage
-    die
+    exit 0
 elif [ "$#" -gt 1 ]; then
     true
 else
-    echo "Invalid arguments provided."
-    usage
-    die
+    invalid "Invalid arguments provided."
 fi
 
 # Learn more at https://www.redhat.com/archives/open-scap-list/2013-July/msg00000.html
 export OSCAP_PROBE_ROOT
-OSCAP_PROBE_ROOT="$(cd "$1"; pwd)"
+OSCAP_PROBE_ROOT="$(cd "$1" && pwd)"
```

Comment:
This doesn't really handle the case when one can't cd into that directory - in that case, the `OSCAP_PROBE_ROOT` will be empty, which is not handled. I would go for calling `die` right afterwards if `OSCAP_PROBE_ROOT` is empty.

Reply:
Added fall-back `die`.